app/Http/Controllers/UserController.php

```diff
@@ -13,7 +18,8 @@ class UserController extends Controller
      */
     public function index()
     {
-        //
+        $users = User::withoutGlobalScope('active')->paginate(config('view.page_size'));
+        return view('users.index', ['users' => $users]);
```

Comment:
```suggestion
        $users = User::withoutGlobalScope('active')->paginate(config('view.page_size'));
        
        return view('users.index', ['users' => $users]);
```
1 dòng trống trc return